int8 motor_id

bool  motor_onoff
int8  motor_desired_direction 
float64 motor_desired_speed